DigitalLearningSolutions.Web/Views/TrackingSystem/Delegates/DelegateProgress/EditCompletionDate.cshtml

```diff
@@ -0,0 +1,57 @@
+@using DigitalLearningSolutions.Web.Models.Enums
+@using DigitalLearningSolutions.Web.ViewComponents
+@using DigitalLearningSolutions.Web.Views.TrackingSystem.Delegates.DelegateProgress
+@using Microsoft.AspNetCore.Mvc.TagHelpers
+@model EditCompletionDateViewModel
+
+@{
+  var errorHasOccurred = !ViewData.ModelState.IsValid;
+  ViewData["Title"] = errorHasOccurred ? "Error: Edit Completion Date" : "Edit Completion Date";
+
+  var routeParamsForBackLink = new Dictionary<string, string?>();
+
+  if (Model.AccessedVia.Equals(DelegateProgressAccessRoute.CourseDelegates)) {
+    routeParamsForBackLink.Add("accessedVia", Model.AccessedVia.ToString());
+    routeParamsForBackLink.Add("progressId", Model.ProgressId.ToString());
+  } else {
+    routeParamsForBackLink.Add("delegateId", Model.DelegateId.ToString());
+  }
+  var exampleDate = DateTime.Today;
+}
+
+<div class="nhsuk-grid-row">
+  <div class="nhsuk-grid-column-full">
+    @if (errorHasOccurred) {
+      <vc:error-summary order-of-property-names="@new []{ nameof(@Model.Day), nameof(@Model.Month), nameof(@Model.Year)}" />
+    }
+
+    <h1 class="nhsuk-heading-xl">Edit completion date for @Model.CourseName</h1>
+
+    <p class="nhsuk-lede-text nhsuk-u-margin-top-4">
+      For example 1 7 2021. Leave the boxes blank to clear the completion date.
+    </p>
+
+    <form class="nhsuk-u-margin-bottom-3" method="post" novalidate>
+       <input type="hidden" asp-for="DelegateId" />
+       <input type="hidden" asp-for="ProgressId" />
+       <vc:date-input id="completion-date"
+                             label="completion date inputs"
```

Comment:
This label needs to be "Completion date". This is displayed on the page:
![image](https://user-images.githubusercontent.com/59561751/143575844-d2d9860e-37da-4e3a-833a-9404449c2e14.png)

Reply:
done